client/src/pages/About/index.jsx

```diff
@@ -1,7 +1,33 @@
 import React from "react";
 
 const About = () => {
-    return <div>inside about</div>;
+    return <div className="bg-red-100"><br></br>
+          <div className=" rounded-lg p-4 mb-4">
+                <h1 className="text-6xl font-bold italic text-center">About Us</h1>
+            </div>
+            <div className=" rounded-lg p-4 mb-4">
+                <h3 className="text-2xl font-semibold text-center">Our Mission</h3><br></br>
+                <p className="text-gray-800 text-center">Drawn2Shoe is a user-friendly e-commerce platform, developed using ReactJS and Node.js. It offers customers the ability to purchase and personalize their shoes. The platform's intuitive interface ensures seamless navigation and usability across a variety of devices.</p>
+            </div>
```

Comment:
Maybe try to keep this non-technical. User don't need to know which stack we are using, but will be happy to know what we are doing :)

Reply:
> Maybe try to keep this non-technical. User don't need to know which stack we are using, but will be happy to know what we are doing :)

Yeah , I did it with general purpose just a simple demo . Surely I will enhance that part